yw_website/yw_website/apps/website/templates/includeable/workflow_list.html

```diff
@@ -21,11 +19,10 @@
 
 {% if workflow_list %}
 
-    <ol id="elementStyle">
-        <script type="text/javascript" src={% static "scripts/home_render_options.js" %}></script>
-
+    <div>
         {% for workflow in workflow_list %}
-            <li>
+        <a href="{% url 'detailed_workflow' workflow.id workflow.version_id %}" style="text-decoration: none; color:inherit;">
```

Comment:
inline styling

Reply:
For some reason this was not working without the inlining. I would like to merge it for now and deal with it later as we present soon.